### ISSUE TITLE
Can we remove `type` column for implementations.adoc?

### ISSUE THREAD

**Issue by jerstlouis** [opengeospatial/ogcapi-tiles#162]
https://github.com/opengeospatial/ogcapi-tiles/blob/cd19f49a0388373e291b98a520dbbf49dede23d3/implementations.adoc?plain=1#L9

Other OGC API (candidate) standards are out of scope, and it does not accurately reflect the supported Standards by the different implementations listed, which are indicated in their conformance declarations.

cc. @ghobona @joanma747 @doublebyte1

**Comment by doublebyte1** [comment]
> https://github.com/opengeospatial/ogcapi-tiles/blob/cd19f49a0388373e291b98a520dbbf49dede23d3/implementations.adoc?plain=1#L9
> 
> Other OGC API (candidate) standards are out of scope, and it does not accurately reflect the supported Standards by the different implementations listed, which are indicated in their conformance declarations, and the other relevant standards also list them in their own implementations.adoc.
> 
> cc. @ghobona @joanma747 @doublebyte1

@jerstlouis agreed. Everything in this repository should be about OGC API - Tiles.

**Comment by ghobona** [comment]
@jerstlouis I too agree.
